cloudify_cli/config/parser_config.py

```diff
@@ -400,24 +401,24 @@ def parser_config():
                         'handler': cfy.plugins.download
                     },
                     'list': {
-                        'help': 'List all the plugins on the Manager',
+                        'help': 'List all plugins currently on the Manager',
                         'handler': cfy.plugins.ls
                     },
                     'delete': {
                         'arguments': {
                             '-p,--plugin-id': plugin_id_argument(
-                                hlp='The plugin id'),
+                                hlp="A plugin's ID"),
```

Comment:
I thought I did. That's why I asked you to check for inconsistencies :) It was late at night. I'll do that right now. `The` actually makes more sense.